src/main/webapp/WEB-INF/web.xml

```diff
@@ -31,4 +31,22 @@
         <servlet-name>registerServlet</servlet-name>
         <url-pattern>/registration</url-pattern>
     </servlet-mapping>
+
+    <servlet>
+        <servlet-name>personalareaServlet</servlet-name>
+        <servlet-class>com.epam.machine.controller.PersonalareaServlet</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>personalareaServlet</servlet-name>
+        <url-pattern>/personalarea</url-pattern>
```

Comment:
I think according to REST the better way to name this URL would be 'persons/{id}/profile' or 'clients/{id}/profile'
https://restfulapi.net/resource-naming/